sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -47,6 +47,11 @@ export interface TryAddOptions {
    * The options to use when creating Spans for tracing.
    */
   tracingOptions?: OperationTracingOptions;
+
+  /**
+   * @deprecated Tracing options have been moved to the `tracingOptions` property.
```

Comment:
So, we'll remove it in the next major?

Reply:
That's my hope. Or perhaps we'll major version when the GA of opentelemetry happens?